tools/scylla-types.cc

```diff
@@ -184,6 +220,18 @@ Arguments: 1 or more serialized values.
 
 $  scylla types validate -t Int32Type b34b62d4
 b34b62d4: VALID - -1286905132
+)"},
+    {"decorate", "decorate (calculate the token of) the partition-key", decorate_handler,
```

Comment:
Maybe we should call it "token" or "tokenof"?

"decorate" makes sense as a by-product of a regular listing (for example, dumping an sstable) - you decorate the regular output with extra information. For example `git log --decorate`. Here this is the primary result, not a byproduct.

Reply:
Ok, fixed in v2.